03/app.js

```diff
@@ -0,0 +1,16 @@
+const buttonsList = document.querySelectorAll('button')
+
+const clickEvent = function (elem, e) {
+	e.preventDefault()
+	elem.innerText = 'clicked'
+	console.log('clicked')
+	elem.removeEventListener('click', handleEvent)
+}
+
+const handleEvent = function (e) {
+	clickEvent(this, e)
+}
+
+buttonsList.forEach(function (button) {
```

Comment:
Tutaj nie trzeba sprawdzać bo mamy tablicę (jeśli jest pusta to nic się nie wydarzy)